hack/boilerplate/check-boilerplate.go

```diff
@@ -78,15 +77,15 @@ func checkBoilerplate(content string) error {
 		return fmt.Errorf("the file is missing a boilerplate")
 	}
 	if index < len(boilerplate) {
-		return errors.New("boilerplate has missing lines")
+		return fmt.Errorf("boilerplate has missing lines")
```

Comment:
I am wondering if the change is unnecessary, since I think both are the same. WDYT?

Reply:
Just want to clean something as I already imported `fmt`. ：）